logger/src/lib.rs

```diff
@@ -0,0 +1,141 @@
+use constants::DRIVER_NAME;
+use cstr::to_widechar_ptr;
+use log::LevelFilter;
+use log4rs::encode::pattern::PatternEncoder;
+use log4rs::Handle;
+use log4rs::{
+    append::rolling_file::{
+        policy::compound::{
+            roll::fixed_window::FixedWindowRoller, trigger::size::SizeTrigger, CompoundPolicy,
+        },
+        RollingFileAppender,
+    },
+    config::{Appender, Config, Root},
+};
```

Comment:
```suggestion
use log4rs::{
    append::rolling_file::{
        policy::compound::{
            roll::fixed_window::FixedWindowRoller, trigger::size::SizeTrigger, CompoundPolicy,
        },
        RollingFileAppender,
    },
    config::{Appender, Config, Root},
    encode::pattern::PatternEncoder,
    Handle,
};
```
[nit] grouping `use` statements